src/main/groovy/au/org/ala/ecodata/reporting/WorksProjectXlsExporter.groovy

```diff
@@ -85,13 +85,23 @@ class WorksProjectXlsExporter extends ProjectExporter {
     }
 
     private void exportProject(Map project) {
+
```

Comment:
The WorksProjectXlsExporter is only used by BioCollect so feel free to just add the NESP stuff into the standard headers rather than behind a check here.  
WorksProjectXlsExporter should never be exporting a MERIT project - if it is currently we need to fix that ASAP.

Reply:
@chrisala Thank you, I was about to ask you of this is shared class between Merit and Biocollect